### PR TITLE
Add Node install/update instructions to prerequisites

### LINKED ISSUE #1081: Document prerequisite of the latest Node LTS
I thought that we had documented this, but apparently we don't.

### PR DESCRIPTION
**Pull request by chalin**
- Closes #1081

Preview: https://deploy-preview-1084--docsydocs.netlify.app/docs/get-started/other-options/#node-get-the-latest-lts-release

### CODE REVIEW

**Comment by emckean** [comment]
LGTM!

**Comment by mikeBoterf** [comment]
By recommending nvm to users, you're introducing another prereq. From nodejs:

here's what I had in my MR:
### Install

> Ensure you have the [latest LTS Version](https://nodejs.org/en/about/releases/)

[Debian and Ubuntu based distributions](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions)

tl;dr

```
# Using Ubuntu
curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
sudo apt-get install -y nodejs

# Using Debian, as root
curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
apt-get install -y nodejs
```

[Enterprise Linux based distributions](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions-1)

tl;dr
```
# As root
curl -fsSL https://rpm.nodesource.com/setup_18.x | bash -

# No root privileges
curl -fsSL https://rpm.nodesource.com/setup_18.x | sudo bash -
```

**Comment by chalin** [comment]
Thanks for those details @mikeBoterf. You might consider sharing those details in a [discussion](https://github.com/google/docsy/discussions) so that others can more easily find them. (For now I'd rather keep such details out of the docs.)

**Comment by mikeBoterf** [comment]
Cool. I’ll definitely do that next time. I agree that a discussion seems to be a more probable place

**Comment by prepstarr** [comment]
does this work on computers running Berkeley Systems Distribution? :)

**Comment by mikeBoterf** [comment]
> does this work on computers running variants of Berkeley Systems Distribution? :) I guess with source code? https://cgit.freebsd.org/ports/tree/www/gohugo/pkg-descr they have the port for Hugo and likely all the components. It would be good to add it to the documentation... guten nacht , auf wiedersehen.. a bientot. if we can get side tracked a little bit if I remember clearly it's BSD day in 5 days...

I’d refer you back to the docs. You can install Hugo from a tar ball.. refer to Hugo docs for that. Docsy is a theme so installing it is dependent on node and other items listed in the docs. So really you should determine if you can install the prerequisites for docsy, then things should work normally, BUT there are no guarantees.
